pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -201,8 +201,18 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	}
 	allErrors = append(allErrors, metrics.ValidateShowHiddenMetricsVersion(kc.ShowHiddenMetricsForVersion)...)
 
+	return utilerrors.NewAggregate(allErrors)
+}
+
+// ValidateKubeletLoggingConfiguration validate kubelet logging part config
+func ValidateKubeletLoggingConfiguration(kc *kubeletconfig.KubeletConfiguration) error {
+	allErrors := []error{}
```

Comment:
Predeclaring list of all errors is not needed as we have one function that returns them.
Let's simplify code here